site/content/docs/manifest/backend-service.md

```diff
@@ -121,6 +121,31 @@ Grace period within which to provide containers time to bootstrap before failed
 
 <div class="separator"></div>
 
+<a id="entrypoint" href="#entrypoint" class="field">`entrypoint`</a> <span class="type">String or Array of Strings</span>  
+Override the default entrypoint in the image. 
+```yaml
+entrypoint: /bin/entrypoint --p1 --p2
+```
+
+```yaml
+entrypoint: ["/bin/entrypoint", "--p1", "--p2"]
+```
```

Comment:
Just to conserve some real-estate:
```suggestion
```yaml
# String version.
entrypoint: "/bin/entrypoint --p1 --p2"
# Alteratively, as an array of strings.
entrypoint: ["/bin/entrypoint", "--p1", "--p2"]
```
```